### PR TITLE
ENH: Setup workflow to fetch task-definition from s3 and deploy

### PR DESCRIPTION
**Pull request by chiragrajk**

### CODE REVIEW

**Review by mariux — COMMENTED** [review]
lgtm

File: .github/workflows/task-definition-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        description: 'Json list of services to be deployed. Example: { "include": [{"SERVICE": "rattle"}] }'
+      cluster:
+        type: string
+        required: true
+        description: The name of the ECS service's cluster.
+      environment:
+        type: string
+        required: true
+        default: staging
+        description: The stach environment of the ECS service.
```
Comment (Odaeus) [inline .github/workflows/task-definition-deploy.yml]:
```suggestion
        description: The stack environment of the ECS service.
```

File: .github/workflows/task-definition-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+        description: The name of the container defined in the containerDefinitions section of the ECS task definition
+        default: app
+      service-name:
+        type: string
+        required: false
+        description: The name of the ECS service to deploy to.
+      services:
+        type: string
+        required: false
+        description: 'Json list of services to be deployed. Example: { "include": [{"SERVICE": "rattle"}] }'
```
Comment (Odaeus) [inline .github/workflows/task-definition-deploy.yml]:
```suggestion
        description: 'JSON list of services to be deployed. Example: { "include": [{"SERVICE": "rattle"}] }'
```

File: .github/workflows/task-definition-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+      
+      - name: Check migration
+        id: check-migration
+        run: |
+          if [ "${{ inputs.skip-migration }}" == "true" ]
+          then
+            echo "::set-output name=command::"
+          else
+            echo "::set-output name=command::${{ inputs.init-task-command }}"
+          fi
```
Comment (Odaeus) [inline .github/workflows/task-definition-deploy.yml]: I guess this doesn't work with the built-in conditionals for some reason?

File: .github/workflows/task-definition-deploy.yml [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+
+jobs:
+  create-deployment:
+    runs-on: ubuntu-latest
+    name: Create Deployment
+    outputs:
+      deployment_id: ${{ steps.create-deployment.outputs.deployment_id }}
+      target_url: ${{ steps.export-status-url.outputs.url }}
+    steps:
+      - name: Export service status url
```
Comment (Odaeus) [inline .github/workflows/task-definition-deploy.yml]:
```suggestion
      - name: Export service status URL
```

**Review by Odaeus — COMMENTED** [review]
Looks impressive!
Afraid I'm only good for typos and English pedantry though as the rest is unknown to me. I don't see anything obviously wrong though 😄

File: .github/workflows/task-definition-deploy.yml (hunk 3 above)
Comment (chiragrajk) [reply .github/workflows/task-definition-deploy.yml]: `
inputs.skip-migration == 'true' && inputs.init-task-command || ''
`

**Pull request merged**